LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2019 Redhat
```

Comment:
```suggestion
   Copyright 2019 Red Hat, Inc.
```

Reply:
🤔 should it be `2021` ?

Reply:
Think it should be from when the repo was created . Thats a best guess.